.github/expired.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '0 */12 * * *'
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          exempt-issue-milestones: 'future,alpha,beta,release'
+          exempt-pr-milestones: 'bugfix,improvement'
+          exempt-all-pr-assignees: true
+          stale-issue-message: 'This issue is stale because it has been open 120 days with no activity. Remove stale label or comment or this will be closed in 15 days.'
+          stale-pr-message: 'This PR is stale because it has been open 120 days with no activity. Remove stale label or comment or this will be closed in 15 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 30 days with no activity.'
+          close-pr-message: 'This PR was closed because it has been stalled for 30 days with no activity.'
+          days-before-issue-stale: 120
+          days-before-pr-stale: 120
+          days-before-issue-close: 30
+          days-before-pr-close: 30
```

Comment:
The owner of this repo doesn't have enough free time to respond to the pulls and issues in time, so they just get closed. Not sure if that is what we want. I also see no use in closing "expired" issues & PRs.